### PR TITLE
Amr/pmt new grapher

### PR DESCRIPTION
**Pull request by aransfor**
PMT now plots live on new grapher if available

### CODE REVIEW

File: lib/servers/NormalPMTFlow/NormalPMTFlow.py [hunk 1]
```diff
@@ -134,10 +134,17 @@ def getOtherListeners(self,c):
...
-        newSet = yield self.dv.new(name, [('t', 'num')], [('KiloCounts/sec','866 ON','num'),('KiloCounts/sec','866 OFF','num'),('KiloCounts/sec','Differential Signal','num')])
+        ds = yield self.dv.new(name, [('t', 'num')], [('KiloCounts/sec','Differential High','num'),
+                                                      ('KiloCounts/sec','Differential Low','num'),('KiloCounts/sec','Differential Signal','num')])
         self.startTime = time.time()
         yield self.addParameters(self.startTime)
-        name = newSet[1]
+        try:
+            self.grapher = yield self.client.grapher
+            print 'success'
+            self.grapher.plot(ds, 'pmt',False)
```
Comment (jayich) [inline lib/servers/NormalPMTFlow/NormalPMTFlow.py]: space.

**Comment by jayich** [comment]
LGTM.

**Comment by jayich** [comment]
Review: @michaelip9228

**Comment by michaelip9228** [comment]
:8ball: :+1:

**Comment by aransfor** [comment]
why is @michaelip9228 faster than @gregllong

**Comment by gregllong** [comment]
LGTM

**Pull request merged**